x-pack/plugins/siem/public/common/components/exceptions/types.ts

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
NOTE: Many of these types will go away when exceptions structure updated and will use those types instead.